src/ML-SNAP/compute_sna_grid.cpp

```diff
@@ -0,0 +1,328 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/ Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#include "compute_sna_grid.h"
+
+#include "atom.h"
+#include "comm.h"
+#include "domain.h"
+#include "error.h"
+#include "force.h"
+#include "memory.h"
+#include "modify.h"
+#include "neigh_list.h"
+#include "neigh_request.h"
+#include "neighbor.h"
+#include "pair.h"
+#include "sna.h"
+#include "tokenizer.h"
+#include "update.h"
+
```

Comment:
The following include files don't seem to be used:
- domain.h
- neighbor.h
- neigh_list.h (see related comment in class definition header)
- neigh_request.h
- pair.h
- tokenizer.h
and thus the include statement should be removed (unless I overlooked something).

Reply:
Done.